projects/aca-content/src/lib/components/files/files.component.ts

```diff
@@ -64,23 +77,41 @@ import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
     PaginationDirective,
     PageLayoutComponent,
     ToolbarComponent,
+    MatIconModule,
+    MatButtonModule,
+    MatDividerModule,
+    SearchAiInputContainerComponent,
     DynamicColumnComponent
   ],
   templateUrl: './files.component.html',
-  encapsulation: ViewEncapsulation.None
+  styleUrls: ['./files.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  selector: 'aca-files'
 })
 export class FilesComponent extends PageComponent implements OnInit, OnDestroy {
   isValidPath = true;
   isAdmin = false;
   selectedNode: NodeEntry;
   queryParams = null;
   showLoader$ = this.store.select(showLoaderSelector);
-  private nodePath: PathElement[];
-
   columns: DocumentListPresetRef[] = [];
   isFilterHeaderActive = false;
 
-  constructor(private contentApi: ContentApiService, private nodeActionsService: NodeActionsService, private route: ActivatedRoute) {
+  private nodePath: PathElement[];
+  private _searchAiInputState: SearchAiInputState = {
```

Comment:
maybe worth putting that into the base `PageComponent` not to copy-paste for each page file

Reply:
good point, thanks, done